components/kyma-environment-broker/cmd/environmentscleanup/app_builder.go

```diff
@@ -0,0 +1,124 @@
+package main
```

Comment:
How do you feel about moving this to `components/kyma-environment-broker/common` or `components/kyma-environment-broker/internal`? The `cmd` dir is usually reserved for executables and this feels more like a library package.
https://github.com/golang-standards/project-layout/tree/509e1be/cmd